USERS.md

```diff
@@ -65,6 +65,12 @@ Users (Alphabetically)
       D: Datadog is using Cilium in AWS (self-hosted k8s)
       U: ENI Networking, Service load-balancing, Encryption
       Q: @lbernail, @roboll
+      
+    * N: GitLab
+      D: GitLab is using Cilium as a method for implementing network policies inside Auto DevOps deployed clusters for GitLab customers using k8s
```

Comment:
Maybe we can shorten this a bit into `GitLab is using Cilium to implement network policies inside Auto DevOps deployed clusters for GitLab customers using k8s`?

Reply:
👍 I will make that change.

Reply:
Updated: https://github.com/cilium/cilium/pull/10485/commits/c03ec7ae0783a4153b814e2ade9de9ea8e79b851